src/logic/dictionaryUtils.ts

```diff
@@ -40,12 +63,13 @@ export function matchesToCommanderHelper(
     playerNameFilter?: string,
     startDate?: Date,
 ): { [id: string]: Commander } {
-    const filteredMatches = filterMatchesByDate(matches, startDate);
+    const filteredMatches = filterMatchesBySeatCount(filterMatchesByDate(matches, startDate), 4);
```

Comment:
This isn't exactly what we want. What we want is to still have all the matches that a player was in to show up in their matches array, but simply not have them count toward their total wins or match count. 

This will remove all matches with less than 4 seats across the board. This is not exactly what we want, since we still want 3 match games to appear in the player's history, but not actually used for any calculations. 

The place you want to apply this check would be where we are applying the wins of the player.

Reply:
TLDR: this hides 3 player matches in the player details match history from being seen. We just want to not include them in math, but not hide them.